detector/aws_elb_invalid_security_group.go

```diff
@@ -10,22 +10,20 @@ import (
 
 type AwsELBInvalidSecurityGroupDetector struct {
 	*Detector
-	IssueType      string
-	TargetType     string
-	Target         string
-	DeepCheck      bool
 	securityGroups map[string]bool
 }
 
 func (d *Detector) CreateAwsELBInvalidSecurityGroupDetector() *AwsELBInvalidSecurityGroupDetector {
-	return &AwsELBInvalidSecurityGroupDetector{
+	nd := &AwsELBInvalidSecurityGroupDetector{
 		Detector:       d,
-		IssueType:      issue.ERROR,
-		TargetType:     "resource",
-		Target:         "aws_elb",
-		DeepCheck:      true,
 		securityGroups: map[string]bool{},
 	}
+	nd.Name = "aws_elb_invalid_security_group"
+	nd.IssueType = issue.ERROR
+	nd.TargetType = "resource"
+	nd.Target = "aws_elb"
```

Comment:
__[gometalinter]__
* [goconst] 3 other occurrence(s) of "aws_elb" found in: aws_elb_duplicate_name.go:23:14 aws_elb_invalid_instance.go:24:14 aws_elb_invalid_subnet.go:24:14

<!-- Comment by [SideCI](https://sideci.com) -->